src/workspaceIndex.jsx

```diff
@@ -56,8 +56,10 @@ async function init() {
 
   // Datadog setup
   if (ddApplicationId && !ddClientToken) {
+    // eslint-disable-next-line no-console
     console.warn('Datadog applicationId is set, but clientToken is missing');
   } else if (!ddApplicationId && ddClientToken) {
+    // eslint-disable-next-line no-console
```

Comment:
I think some of these console prints related to DD RUM are ligit to have, so maybe considering to ignore them from eslint.rc?

but this is a very minor point